addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13625,15 +13628,15 @@ msgid "Is Apple's Bonjour service installed? See log for more info."
 msgstr ""
 
 msgctxt "#34302"
-msgid "AirPlay requires Zeroconf to be enabled."
+msgid "AirPlay requires \"Zeroconf\" to be enabled."
 msgstr ""
 
 msgctxt "#34303"
-msgid "Unable to stop Zeroconf"
+msgid "Unable to stop \"Zeroconf\""
```

Comment:
Only reason for doing is, it is really zeroconf on all external references I found, but it is widely used Zeroconf, so ok not quotes here.